### PR TITLE
Matt wednesday

### PR DESCRIPTION
**Pull request by Snobeard**

### CODE REVIEW

File: css/journal.css [hunk 1]
```diff
@@ -130,20 +130,29 @@ button:focus {
...
     font-size: 2.3em;
     color: white;
     font-family: Roboto;
     margin-bottom: .7em;
-    width: 96%;
+    width: 94%;
     display: inline-block;
     overflow-wrap: break-word;
 }
+#journalEntriesGoHere section h5 {
```
Comment (Snobeard) [inline css/journal.css:142]: <b>added detail about which workout was selected when journal was written</b>

File: css/style.css [hunk 2]
```diff
@@ -147,14 +148,6 @@ main h2 {
   text-shadow: 2px 2px 5px #272727;
 }
```
Comment (Snobeard) [inline css/style.css:150]: <b>extraneous h2 style

File: css/style.css [hunk 3]
```diff
@@ -297,3 +290,74 @@ footer ul li {
   margin: 0 auto;
   padding-right: .7em;
 }
+
```
Comment (Snobeard) [inline css/style.css:301]: <b>journal styling for index.html

File: js/app.js [hunk 4]
```diff
@@ -1,12 +1,12 @@
 'use strict';
 
 var userName = document.getElementById('name');
-if (localStorage.workout) {
-  userName.value = JSON.parse(localStorage.workout.userName);
-}
 var workoutSelection = document.getElementById('workoutSelection');
 Workouts.allWorkouts = [];
```
Comment (Snobeard) [inline js/app.js:6]: <b>this was the small error that wasn't producing the name value

File: js/app.js [hunk 5]
```diff
@@ -48,20 +48,20 @@ Workouts.prototype.userInput = function(event){
...
   var cize = new Workouts('CIZE', [2], [2], ['dance'], '<h4>Choose Your Workout</h4><img class="left" id="workoutLogo" src="img/cize.jpg" alt="BeachBody logo"></img><p id="workoutText">CIZE™ breaks down the hottest professionally choreographed dance courses, step-by-step, so that anyone can dance. You\'ll be so focused on learning the moves that you won\'t even realize you\'re burning calories and losing weight!</p>');
   var dayFixExtreme	= new Workouts('21 Day Fix EXTREME', [2], [3], ['extreme'], '<h4>Choose Your Workout</h4><img class="left" id="workoutLogo" src="img/21dayfixextreme.jpg" alt="BeachBody logo"></img><p id="workoutText">Celebrity trainer Autumn Calabrese has combined simple portion control, clean eating, and extreme 30-minute workouts to help you get the hardbody you’ve always wanted.</p>');
```
Comment (Snobeard) [inline js/app.js:50]: <b>some spacing for better visuals on journal page

File: js/index.js [hunk 6]
```diff
@@ -0,0 +1,73 @@
+'use strict';
```
Comment (Snobeard) [inline js/index.js:1]: <b>journal things

File: js/journal.js [hunk 7]
```diff
@@ -78,7 +84,7 @@ function changeJournalHeader() {
   title.textContent = (JSON.parse(localStorage.workout).name);
 }
```
Comment (Snobeard) [inline js/journal.js:86]: <b>linter doesn't like me =[

**Review by Nick-Gibson — APPROVED** [review]

**Pull request merged**